usagov/vote-gov-drupal
# VOTE-1177 remove static html directory from git

Fix view permission issue
Update lando static command

## Jira ticket (required)
https://bixal-projects.atlassian.net/browse/VOTE-1177

## Description (optional)
This fixes issues with the Tome static site export and removes the files from git tracking.

## Deployment and testing (required, if applicable)
### Post-deploy steps
1. run `lando retune`
2. run `lando drush pm-uninstall migrate -y`
3. run `lando static`

### Testing steps
1. Make sure the static files compiled successfully without error
2. Visit the site at http://html.vote-gov.lndo.site
